src/nodes.coffee

```diff
@@ -1113,6 +1115,7 @@ exports.IdentifierLiteral = class IdentifierLiteral extends Literal
   astProperties: ->
     return
       name: @value
+      declaration: !!@isDeclaration
```

Comment:
Any `IdentifierLiteral` node class objects that represent a variable declaration will have had their `isDeclaration` property set before AST generation, so here we expose that under a `declaration` field on `Identifier` AST nodes (this represents an extension to the Babel-style `Identifier` node, since in JS there are separate AST nodes for the variable declarations)